src/AsyncTimer.cpp

```diff
@@ -73,6 +73,14 @@ void AsyncTimer::cancelAll(bool includeIntervals) {
   }
 }
 
+unsigned long AsyncTimer::getRemaining(unsigned short id) {
+  unsigned long now = millis();

```

Comment:
Since `now` is only used once, it should be omitted and replaced by `millis()` call